install.lua

```diff
@@ -31,12 +31,12 @@ end
 
 local function rewriteDofile(filename, required)
```

Comment:
Do we even need the `required` argument, or should this just blindly replace every dofile?